### PR TITLE
doc(styles): Scroll menu and main content separately

### LINKED ISSUE #1682: Main content scrolls with menu items
## Describe the bug
Main content scrolls with menu items which makes up bad user experience for browsing documentation

## To Reproduce
1. Visit https://porter.sh/ on mobile device
2. Click on any item like quick start
3. Click menu and scroll
4. See scrolling issue

## Expected behavior
Only menu should scroll

## Porter Command and Output
https://user-images.githubusercontent.com/51229945/126891669-275f151e-4eb6-490d-a835-9b51cc47feb9.mp4

### PR DESCRIPTION
**Pull request by thisisommore**
# What does this change
Menu now scroll separately and finitely

https://user-images.githubusercontent.com/51229945/126903548-a8301981-1f28-4ada-a1ee-82f49d7c934c.mp4

# What issue does it fix
Closes #1682 

# Notes for the reviewer
In desktop the box shadow of `<a class="exit-off-canvas"></a>` interfaces ui of scroll bar
porter-screenshot
There are three options to fix this,
- Use custom scroll bar style
- Remove `<a class="exit-off-canvas"></a>` completly
-  Remove box shadow property, possibly this https://github.com/getporter/porter/blob/05332860d3ad92a05d30f740165e7609106c6c4e/docs/themes/porter/assets/sass/foundation/components/_offcanvas.scss#L280

Due to `<a class="exit-off-canvas"></a>` in my changes home page doesn't scroll when menu is open.

### CODE REVIEW

**Comment by thisisommore** [comment]
@DARK-art108 @carolynvs would like to discuss this ?

**Comment by DARK-art108** [comment]
Awesome @thisisommore Looks good to me!

**Review by DARK-art108 — APPROVED** [review]
Looks good to merge!

**Comment by DARK-art108** [comment]
@thisisommore can you elaborate more on what you have **Note for the reviewer** as I see the desktop site is working fine after this fix!

**Comment by thisisommore** [comment]
> @thisisommore can you elaborate more on what you have **Note for the reviewer** as I see the desktop site is working fine after this fix!

@DARK-art108 
Sorry if I was being unclear there,
The issue here is minor and is only visible when windows width is small (thats when left hidden menu is available) and can be ignored.
After making menu scrollable it gets it own scrollbar, and that scrollbar gets under the shadow of the element which is in right side.
For reference see this screenshot
image
and this screenshot after removing the element in side right which was causing shadow,
image

**Review by carolynvs — CHANGES_REQUESTED** [review]
Thanks for fixing this!

I tested the site on a small screen, and can see the problem that you highlighted with scrolling when the menu is open. I tried removing the exist-off-canvas and that seemed to fix it without any side-effects. 

Can you remove that element as well?

**Comment by thisisommore** [comment]
> Thanks for fixing this!
> 
> I tested the site on a small screen, and can see the problem that you highlighted with scrolling when the menu is open. I tried removing the exist-off-canvas and that seemed to fix it without any side-effects. 
> 
> Can you remove that element as well?

I can remove that element by tomorrow and thanks for checking this out 😀, between I see that you have self assigned this, did you already removed element ?

**Comment by carolynvs** [comment]
We use the assignment field on pull requests to help track which maintainer is "managing" a pull request, it doesn't mean that we are taking over an issue or anything. 😀

**Comment by thisisommore** [comment]
> We use the assignment field on pull requests to help track which maintainer is "managing" a pull request, it doesn't mean that we are taking over an issue or anything. 😀 

Thanks, got it.

**Comment by thisisommore** [comment]
I have removed the element from where it was causing issue, I didn't removed it from all html files since I am not aware of issues that might occur by removing the element from every file.

**Comment by DARK-art108** [comment]
@thisisommore it's working fine now in small screens, now site look more awesome!

**Comment by thisisommore** [comment]
Just noticed, we are having issues with scroll in side menu.
Reference https://developers.google.com/web/updates/2016/12/url-bar-resizing
I will try to fix this.

**Comment by thisisommore** [comment]
Now even more stable.
Chrome url bar now won't cause issues.
https://user-images.githubusercontent.com/51229945/127325805-ad3e41a2-0424-48f2-9642-79feff617439.mp4

**Review by DARK-art108 — APPROVED** [review]
Looks fine to me!

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -54,3 +54,4 @@ and we will add you. **All** contributors belong here. 💯
 * [Divyam Bhasin](https://github.com/divbhasin)
 * [Ritesh Yadav](https://github.com/DARK-art108)
 * [Jérémy Audiger](https://github.com/jaudiger)
+* [Om More](https://github.com/thisisommore)
```
Comment (carolynvs) [inline CONTRIBUTORS.md:57]: Thank you for adding your name to our contributors page! 🎉

**Review by carolynvs — APPROVED** [review]
This looks great. I really appreciate you contributing some much needed CSS and design skills to our site. 👍

**Pull request merged**